manifests/emissary/emissary-emissaryns.yaml.in

```diff
@@ -407,113 +390,129 @@ spec:
       allow_non_local: true
       enabled: false
 ---
-apiVersion: v1
-kind: ServiceAccount
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
 metadata:
   labels:
     app.kubernetes.io/instance: emissary-ingress
     app.kubernetes.io/managed-by: getambassador.io
-    app.kubernetes.io/name: emissary-ingress-agent
+    app.kubernetes.io/name: emissary-ingress
     app.kubernetes.io/part-of: emissary-ingress
     product: aes
-  name: emissary-ingress-agent
-  namespace: emissary
+    rbac.getambassador.io/role-group: emissary-ingress
+  name: emissary-ingress-apiext
+  namespace: emissary-system
+rules:
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - get
+  - list
+  - watch
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
+kind: RoleBinding
 metadata:
   labels:
     app.kubernetes.io/instance: emissary-ingress
     app.kubernetes.io/managed-by: getambassador.io
-    app.kubernetes.io/name: emissary-ingress-agent
+    app.kubernetes.io/name: emissary-ingress
     app.kubernetes.io/part-of: emissary-ingress
     product: aes
-  name: emissary-ingress-agent
+  name: emissary-ingress-apiext
+  namespace: emissary-system
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: emissary-ingress-agent
+  kind: Role
+  name: emissary-ingress-apiext
 subjects:
 - kind: ServiceAccount
-  name: emissary-ingress-agent
+  name: emissary-ingress
   namespace: emissary
 ---
-aggregationRule:
-  clusterRoleSelectors:
-  - matchLabels:
-      rbac.getambassador.io/role-group: emissary-ingress-agent
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   labels:
     app.kubernetes.io/instance: emissary-ingress
-    app.kubernetes.io/managed-by: getambassador.io
-    app.kubernetes.io/name: emissary-ingress-agent
-    app.kubernetes.io/part-of: emissary-ingress
-    product: aes
-  name: emissary-ingress-agent
-rules: []
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: agent
+    app.kubernetes.io/version: 1.0.16
+    helm.sh/chart: agent-1.0.16
+    rbac.getambassador.io/role-group: emissary-ingress-agent
+  name: emissary-ingress-agent-applications
+rules:
+- apiGroups:
+  - argoproj.io
+  resources:
+  - applications
+  verbs:
+  - get
+  - list
+  - watch
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   labels:
     app.kubernetes.io/instance: emissary-ingress
-    app.kubernetes.io/managed-by: getambassador.io
-    app.kubernetes.io/name: emissary-ingress-agent
-    app.kubernetes.io/part-of: emissary-ingress
-    product: aes
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: agent
+    app.kubernetes.io/version: 1.0.16
+    helm.sh/chart: agent-1.0.16
     rbac.getambassador.io/role-group: emissary-ingress-agent
-  name: emissary-ingress-agent-pods
+  name: emissary-ingress-agent-default-ns
```

Comment:
This seems wrong, we shouldn't have any references to the default namespace in this templated yaml, correct?

Reply:
Good catch... Looks like it's actually an issue with the imported agent template (see [here](https://github.com/datawire/ambassador-agent/blob/d0b774d48acb780c1a5604eaaad9eb9964b96914/helm/ambassador-agent/templates/clusterrole.yaml#L57)). 

Not sure what the best course of action is. Should we try to fix the agent side?

Reply:
Maybe ask Jake/Kevin to see if there is something with Telepresence.

Reply:
After reaching out to the Telepresence team, it seems this is used to generate the cluster_id. We can keep this in for now as it won't affect anything on our end.